package.json

```
@@ -68,7 +68,7 @@
     "body-parser-xml": "2.0.5",
     "dateformat": "3.0.3",
     "express": "4.18.1",
-    "iotagent-node-lib": "https://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "https://github.com/Keshav-NEC/iotagent-node-lib.git#Fix_for_1569",
```

Now PR https://github.com/telefonicaid/iotagent-node-lib/pull/1571 has been merged, it's time to revert this change :)

```suggestion
    "iotagent-node-lib": "https://github.com/telefonicaid/iotagent-node-lib.git#master",
```

---

Changes done as per comment received.